sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/SaplingParamTool.kt

```diff
@@ -17,9 +17,6 @@ import java.nio.channels.Channels
 // TODO [#665]: https://github.com/zcash/zcash-android-wallet-sdk/issues/665
 // TODO [#665]: Recover from corrupted sapling-spend.params and sapling-output.params
 
-// TODO [#664]: https://github.com/zcash/zcash-android-wallet-sdk/issues/664
-// TODO [#664]: Check size of download for sapling-spend.params and sapling-output.params
-
 // TODO [#611]: https://github.com/zcash/zcash-android-wallet-sdk/issues/611
 // TODO [#611]: Move Params Directory to No Backup Directory
 
```

Comment:
Pre-existing issue, but I realize looking at this now that `SaplingParamTool` could be made `object` and the `companion` can then go away.

Reply:
Yep, good point, done.